apps/dav/lib/CalDAV/Reminder/NotificationProvider/EmailProvider.php

```diff
@@ -196,7 +205,7 @@ private function getOrganizerEMailAndNameFromEvent(VEvent $vevent):?array {
 		}
 
 		$organizer = $vevent->ORGANIZER;
-		if (strcasecmp($organizer->getValue(), 'mailto:') !== 0) {
+		if (!str_starts_with($organizer->getValue(), 'mailto:')) {
```

Comment:
This is PHP 8 exclusive, so we can't use this yet.
```suggestion
		if (strcasecmp($organizer->getValue(), 'mailto:') !== 0) {
```

Reply:
:+1: